accounts/views.py

```diff
@@ -237,7 +237,7 @@ def send_activation(user):
         'username': username,
         'hash': uid_hash
     }
-    send_mail_template(u'Your activation link.', 'accounts/email_activation.txt', tvars, user_to=user)
```

Comment:
interesting, I didn't see that some of our subjects had . at the end. I see you've removed them, good.